### PR TITLE
Desktop cleanup

### PR DESCRIPTION
**Pull request by mdescalzo**
* Forces links to load in external browsers.
* Fixed the bogus push notifications disabled message in settings.
* Removed the 'PROTOTYPE' subheading from header in electron builds.

### CODE REVIEW

File: app/views/settings.js [hunk 1]
```diff
@@ -24,7 +24,7 @@
                 notificationSoundMuted: await F.state.get("notificationSoundMuted"),
                 allowBugReporting: !(await F.state.get("disableBugReporting")),
                 allowUsageReporting: !(await F.state.get("disableUsageReporting")),
-                hasPushNotifications: !!(await F.state.get('serverGcmHash')),
+                hasPushNotifications: !!(await F.state.get('serverGcmHash')) || (F.electron && self.Notification && Notification.permission),
```
Comment (mayfield) [inline app/views/settings.js]: Hmmm..  This isn't exactly true.  Could we just filter the push notification thing out of the template by wrapping it in {{#if isElectron}} .... {{/if}} ?

Comment (mdescalzo) [reply app/views/settings.js]: Done and pushed.

File: app/views/settings.js [hunk 2]
```diff
@@ -25,6 +25,7 @@
                 allowBugReporting: !(await F.state.get("disableBugReporting")),
                 allowUsageReporting: !(await F.state.get("disableUsageReporting")),
                 hasPushNotifications: !!(await F.state.get('serverGcmHash')),
+                isElectron: (F.electron),
```
Comment (mayfield) [inline app/views/settings.js]: The parans are a noop, but what you might want (but not strictly required) is a `!!F.electron` which coerces the F.electron truthiness to a boolean.

File: templates/views/settings.html [hunk 3]
```diff
@@ -110,6 +110,8 @@
             {{/if}}
         </div>
 
+        {{#if isElectron}}
```
Comment (mayfield) [inline templates/views/settings.html]: You can also use `{{unless isElectron}}` which is more idiomatic.

File: electron/renderer.js [hunk 4]
```diff
@@ -3,7 +3,18 @@
 const {ipcRenderer} = require('electron');
 
 self.F = self.F || {};
-const ns = self.F.electron = {};
+const ns = self.F.electron = {
+    isElectron: true
```
Comment (mayfield) [inline electron/renderer.js]: I think this is redundant, no?  presence of F.electron alone indicates that you're in it.

**Pull request merged**